code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -83,6 +91,10 @@
 		return
 	return ..()
 
+/obj/structure/closet/crate/toggle(mob/user, by_hand = FALSE)
+	if(!(opened ? close() : open(by_hand)))
+		to_chat(user, "<span class='notice'>It won't budge!</span>")
```

Comment:
too much punctuation to comfortably parse
```suggestion
	var/success = opened ? close() : open(by_hand)
	if(!success)
		to_chat(user, "<span class='notice'>It won't budge!</span>")
```

Reply:
I'm duplicating the closet toggle proc mostly, just to add an argument and I'd rather not change it.

Reply:
Alright, fair.